packages/stories/src/index.tsx

```diff
@@ -33,6 +34,25 @@ export default function Stories({
   const hasCalledEndedCb = useRef<any>(false);
   const hasCalledStartedCb = useRef<any>(false);
 
+  // Preload stories  
+  const preloadedStoriesIndexs: number[] = useMemo(() => [], []);
```

Comment:
How to use useRef in this case? is not clear for me.

Reply:
```
// create a simple ref with default value as empty array;
  const preloadedStoriesIndexsRef  = useRef([])
 
 // where ever u want to use its value u can do it by 
 console.log(preloadedStoriesIndexsRef.current);
 
 // if u want to update the value . then do simple push 
preloadedStoriesIndexsRef.current.push(valueToBeInRef);

// adding a value will not trigger a re render because its a ref. 
// also the ref value will always be in sync as react will guarantee it. 
// alll operations on ref are synchronous too. 

```